### PR TITLE
stats: append proxy config extra stat tags

### PR DESCRIPTION
**Pull request by kyessenov**
Organize the telemetry extension tags in the bootstrap.
Add `request_operation` pre-configured label.
Append proxy config tags to the defaults and the annotation tags.

### CODE REVIEW

File: pkg/bootstrap/config.go [hunk 1]
```diff
@@ -201,11 +237,25 @@ func getStatsOptions(meta *model.NodeMetadata, nodeIPs []string) []option.Instan
 		return substituteValues(inclusionOption, "{pod_ip}", nodeIPs)
 	}
 
+	extraStatTags := make([]string, 0, len(DefaultStatTags))
+	extraStatTags = append(extraStatTags,
```
Comment (howardjohn) [inline pkg/bootstrap/config.go:241]: does order matter?

Comment (howardjohn) [reply pkg/bootstrap/config.go:241]: and are duplicates ok?

Comment (kyessenov) [reply pkg/bootstrap/config.go:241]: Order does not matter. Duplicates don't matter unless we somehow produce duplicates and that would be a bug.

Comment (howardjohn) [reply pkg/bootstrap/config.go:241]: what is a user puts a tag that is already in the default?

File: pilot/cmd/pilot-agent/config_test.go [hunk 2]
```diff
@@ -90,12 +91,14 @@ discoveryAddress: annotation:123
...
   ANNOTATION: something
 drainDuration: 5s
+extraStatTags: ["b"]
 `,
 			expect: func() meshconfig.ProxyConfig {
 				m := mesh.DefaultProxyConfig()
 				m.DiscoveryAddress = "annotation:123"
 				m.ProxyMetadata = map[string]string{"ANNOTATION": "something"}
 				m.DrainDuration = types.DurationProto(5 * time.Second)
+				m.ExtraStatTags = []string{"b"}
```
Comment (howardjohn) [inline pilot/cmd/pilot-agent/config_test.go:101]: thanks!

File: pkg/bootstrap/config.go (hunk 1 above)
Comment (kyessenov) [reply pkg/bootstrap/config.go:241]: That would be strange but harmless.

**Pull request merged**